content/en/docs/setup/best-practices/node-conformance.md

```diff
@@ -1,7 +1,8 @@
 ---
```

Comment:
(possibly in a future PR)
When we're informing the reader about best practices for nodes, we could signpost them to https://kubernetes.io/docs/tasks/debug-application-cluster/monitor-node-health/ and / or similar tooling.

I don't think production cluster operators should make sure that they using conformance tests, as described here, to validate that newly-added Nodes are healthy. Given the caveats mentioned, it seems like a bad idea to do that.

Reply:
Yes, I agree this is a good point. I will raise an issue to refactor after we okay the walk-through of the left nav for Getting Started.